src/main/scala/treadle/TreadleTester.scala

```diff
@@ -433,6 +449,14 @@ object TreadleTester {
     * @return
     */
   def apply(input : String, optionsManager: HasTreadleSuite = getDefaultManager): TreadleTester = {
-    new TreadleTester(input, optionsManager)
+    val sourceAnnotation = FirrtlSourceAnnotation(input)
+    TreadleTester(sourceAnnotation +: optionsManager.toAnnotationSeq)
+  }
+
+  def apply(annotations: AnnotationSeq): TreadleTester = {
```

Comment:
While it does not currently do this, I think that `TransformLike[A]` will, in the future, extend `(A) => A`. Can you use a different method, other than apply here? Alternatively, if you do use an apply, it should be `(AnnotationSeq) => AnnotationSeq`.